content/en/security/cloud_security_management/guide/public-accessibility-logic.md

```diff
@@ -14,22 +14,31 @@ Datadog uses a graph processing framework to map relationships between cloud res
 
 For more information on network reachability, see the [AWS documentation][34] and the [AWS Network Reachability Analyser][35]. Currently, the `Is Publicly Accessible` facet is only available for AWS resources.
 
+This page details on a resource-by-resource basis the logic behind public accessibility determination for a resource.
+
+**Note:** Not all resources are able to be classified as publicly accessible or not publicly accessible through only their resource configuration. Such resources are not included here.
+
 ## Resource dependency graph
 
-The following diagram shows how related resources are used to determine whether other resources are publicly accessible. For example, a CloudTrail Trail stored in a public S3 bucket is itself publicly accessible. If a resource is publicly accessible because of another resource, the relationship is shown in the Cloud Security Management Misconfigurations Resource relationships graph.
+The following diagrams show how related resources are used to determine whether other resources are publicly accessible. For example, a CloudTrail Trail stored in a public S3 bucket is itself publicly accessible. If a resource is publicly accessible because of another resource, the relationship is shown in the Cloud Security Management Misconfigurations Resource relationships graph.
 
 Use this diagram as a reference for how resources are correlated to determine public accessibility. 
 
-**Note**: Not all resources with the Publicly Accessible attribute are shown in this diagram.
+**Note**: Not all resources with the Publicly Accessible attribute are shown in these diagrams.
 
-{{< img src="security/cloud_security_management/guide/public_accessibility_relationships.png" alt="A graph diagram showing the relationships between resources that are used to determine public accessibility" width="100%">}}
+### AWS
 
-## Public accessibility logic by resource
+{{< img src="security/cloud_security_management/guide/public_accessibility_relationships_aws.png" alt="A graph diagram showing the relationships between resources that are used to determine public accessibility for AWS" width="100%">}}
 
-The following section details on a resource-by-resource basis the logic behind public accessibility determination for a resource.
+### Azure
 
-**Note:** Not all resources are able to be classified as publicly accessible or not publicly accessible through only their resource configuration. Such resources are not included here.
+{{< img src="security/cloud_security_management/guide/public_accessibility_relationships_azure.png" alt="A graph diagram showing the relationships between resources that are used to determine public accessibility for Azure" width="70%">}}
+
+### GCP
```

Comment:
We're supposed to call it "Google Cloud" now, but not sure what's our stance on the docs side, I'll defer to someone from the documentation team

Reply:
Agreed. We should change this to "Google Cloud".